runtime/doc/api.txt

```diff
@@ -3194,9 +3194,11 @@ nvim_open_win({buffer}, {enter}, {config})                   *nvim_open_win()*
                   • col: Column position in units of "screen cell width", may
                     be fractional.
                   • focusable: Enable focus by user actions (wincmds, mouse
-                    events). Defaults to true. Non-focusable windows can be
-                    entered by |nvim_set_current_win()|, or, when the `mouse`
-                    field is set to true, by mouse events.
+                    events). Defaults to true. Non-focusable windows are
+                    considered to be UI elements which Nvim may treat
```

Comment:
This declaration is a bit too strong. For now let's be coy and just say the technical differences, without going as far as "we consider these UI elements". Because generally, windows are windows.

Since this PR was created, I added `:help focusable`. So let's mention the `:tabs` behavior there, and link to `|focusable|` from here.